OpenRA.Mods.Common/Scripting/Global/ReinforcementsGlobal.cs

```diff
@@ -137,7 +137,7 @@ public Actor[] Reinforce(Player owner, string[] actorTypes, CPos[] entryPath, in
 			}
 			else
 			{
-				var heli = transport.TraitOrDefault<Helicopter>();
+				var heli = transport.TraitOrDefault<Aircraft>();
```

Comment:
If this has an implicit dependency on heli logic I see no reason to rename the variable.
`heli` is a hint while `aircraft` is just repeating the type for consistency.